homeassistant/components/template/light.py

```diff
@@ -371,6 +371,18 @@ async def async_turn_on(self, **kwargs):
                 kwargs[ATTR_COLOR_TEMP],
             )
             self._temperature = kwargs[ATTR_COLOR_TEMP]
+            if self._color_template is None:
+                self._color = None
+            optimistic_set = True
+
+        if self._color_template is None and ATTR_HS_COLOR in kwargs:
+            _LOGGER.info(
```

Comment:
```suggestion
            _LOGGER.debug(
```

Reply:
This copies the existing prints which are also severity `info`. I agree `debug` is better. OK to change them all to `debug` in this PR?

Reply:
Yeah, that's ok.